modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -151,6 +152,19 @@ export default class MVTLayer extends TileLayer {
     return super.onHover(info, pickingEvent);
   }
 
+  getPickingInfo({info, sourceLayer}) {
+    info.sourceLayer = sourceLayer;
+    info.tile = sourceLayer.props.tile;
+
+    const isWGS84 = this.context.viewport.resolution;
+
+    if (!isWGS84 && info.object) {
+      info = {...info, object: transformTileCoordsToWGS84(info, this.context.viewport)};
```

Comment:
Just pass `object` and `tile` separately to `transformTileCoordsToWGS84` so that you don't have to clone the info object.